libs/langchain/langchain/tools/e2b_data_analysis/tool.py

```diff
@@ -97,7 +97,7 @@ class E2BDataAnalysisTool(BaseTool):
     name = "e2b_data_analysis"
     args_schema: Type[BaseModel] = E2BDataAnalysisToolArguments
     session: Any
-    uploaded_files: List[UploadedFile] = Field(default_factory=list)
+    _uploaded_files: List[UploadedFile] = PrivateAttr(default_factory=list)
```

Comment:
think @mlejva just changed this to be public in #12345

Reply:
I think keeping them private as @jakubno changed it is probably the way to go. We manipulate the files internally and expose only the description of all uploaded files.